src/pages/AccountRequest/accountUtil.tsx

```diff
@@ -0,0 +1,55 @@
+import { IFormOption } from "components/Form/interfaces";
+
+export interface IUserFormValues {
+  username: string;
+  email: string;
+  full_name: string;
+  role_id: number;
+  introduction: string;
+  institution_id: number;
+}
+
+export enum EmailPreference {
```

Comment:
Do you use this enum in your code? If so, it may be redundant since it is already included in the `userUtils`.

To streamline your usage, it's recommended to extract or move this enum and any related duplicate code into a shared `util/interfaces.ts` file.